src/sentry/apidocs/parameters.py

```diff
@@ -144,8 +144,8 @@ class CURSOR_QUERY_PARAM(serializers.Serializer):  # type: ignore
 
 
 class MONITOR_PARAMS:
-    MONITOR_ID = OpenApiParameter(
-        name="monitor_id",
+    MONITOR_SLUG = OpenApiParameter(
+        name="monitor_slug",
         location="path",
         required=True,
         type=OpenApiTypes.UUID,
```

Comment:
Should this change? Slugs aren't always UUID now.

Reply:
Yes it should